docs/technical-guide/getting-started/unofficial-options.md

```diff
@@ -9,6 +9,7 @@ There are some other options, **NOT SUPPORTED BY PENPOT**:
 * Install with <a href="https://community.penpot.app/t/how-to-develop-penpot-with-podman-penpotman/2113" target="_blank">Podman</a> instead of Docker.
 * Try the under development <a href="https://github.com/author-more/penpot-desktop/releases/latest" target="_blank">Penpot Desktop app</a>.
 * Try a simple Kubernetes Deployment option <a href="https://github.com/degola/penpot-kubernetes" target="_blank">penpot-kubernetes</a>.
+* Penpot is available <a href="https://apps.yunohost.org/app/penpot">in the catalog</a> of apps installable on YunoHost instances.
```

Comment:
```suggestion
* Penpot is available <a href="https://apps.yunohost.org/app/penpot">in the catalog</a> of apps installable on <a href="https://yunohost.org/">YunoHost</a> instances.
```

I think it would be nice to include a link to the YunoHost landing page to explain what it is to people that don't already know

Reply:
I was considering that initially but decided against it to avoid link spam, especially given catalog page links to both landing and instructions on how to via 'Install with YNH' button.

But I'm OK with linking YNH landing as long as Penpot maintainers don't consider it excessive 👍